src/ebay-inline-notice/inline-notice.tsx

```diff
@@ -6,7 +6,7 @@ import { findComponent } from '../common/component-utils'
 import { EbayIcon, Icon } from '../ebay-icon'
 import { NoticeStatus } from './types'
 
-type Props = {
+type Props = React.HTMLProps<HTMLDivElement> & {
```

Comment:
```suggestion
type Props = React.ComponentProps<'div'> & {
```

Reply:
@darkwebdev could you give a reason why to make this change just to understand. We need to have a consistency between components, (the HTMLProps was inspired by [page-notice](https://github.com/eBay/ebayui-core-react/blob/main/src/ebay-page-notice/page-notice.tsx#L6)). Also all the notice utils uses it HTMLProps